packages/@react-aria/tooltip/src/useTooltipTrigger.ts

```diff
@@ -63,28 +65,39 @@ export function useTooltipTrigger(props: TooltipTriggerProps, state: TooltipTrig
   }, [ref, state]);
 
   let onHoverStart = () => {
-    isHovered.current = true;
+    if (getInteractionModality() === 'pointer') {
```

Comment:
I'm curious, what is the interaction modality returned if the user has a stylus and hovers over a tooltip trigger? I think that can be done via holding the stylus point above the screen (almost touching the screen but not quite). Does it even count as a hover?

Not a blocker, wish I had my old Note phone that had the stylus pen to test this :(

Reply:
oo, i have no stylus pen :( so I do not know

Reply:
how does this case happen? when would `onHoverStart` be called when the interaction modality is *not* pointer? Just a stylus? also could you use `e.pointerType` here instead?

Reply:
this case is for the multiple tooltips stories, steps are
hover first one, leave mouse there
tab to first one
tab to second
tab to third

without this check, the first buttons tooltip shows up again (instead of 3) because when the tooltip from button two disappears, button one gets hovered again (the tooltip from button 2 was blocking it)

Reply:
because of this, i do actually need to check the modality, i can't just use the event pointer type